charts/tfy-gpu-operator/values.yaml

```diff
@@ -76,6 +83,25 @@ aws-eks-gpu-operator:
         value: volume-mounts
       - name: DEVICE_ID_STRATEGY
         value: index
+      - name: MIG_STRATEGY
+        value: mixed
+      - name: NVIDIA_MIG_MONITOR_DEVICES
+        value: all
+    ## @skip aws-eks-gpu-operator.devicePlugin.config
+    config:
+      data:
+        all: ''
+        time-sliced-10: |-
+          version: v1
+          sharing:
+            timeSlicing:
+              renameByDefault: true
```

Comment:
Can we control what it gets renamed to?

Reply:
No: https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/gpu-sharing.html#configuration
It is nvidia.com/gpu.shared